gef.py

```diff
@@ -3619,21 +3635,28 @@ def parse_string_range(s):
 
 @lru_cache()
 def gef_get_auxiliary_values():
-    """Retrieves the auxiliary values of the current execution. Returns None if not running, or a dict()
-    of values."""
+    """Retrieves the ELF auxiliary values of the current execution. This information is provided by
+    the operating system to transfer some kernel level information to the user process. Returns
+    None is not running, or a dict() of values as: {aux_vect_name: int(aux_vect_value)}."""
     if not is_alive():
         return None
 
-    res = {}
-    for line in gdb.execute("info auxv", to_string=True).splitlines():
-        tmp = line.split()
-        _type = tmp[1]
-        if _type in ("AT_PLATFORM", "AT_EXECFN"):
-            idx = line[:-1].rfind('"') - 1
-            tmp = line[:idx].split()
-
-        res[_type] = int(tmp[-1], base=0)
-    return res
+    try:
```

Comment:
This is not easy to read I feel. There are 2 distinct cases which allow to write something better like

```suggestion
    __auxiliary_vector = {}
    for line in gdb.execute("info auxv", to_string=True).splitlines():
        line = line.split('"')[0].strip() # remove the ending string (if any)
        line = line.split() # split the string by space(s)
        __av_type = line[1]
        __av_value = line[:-1]
        _auxiliary_vector[__av_type ] = int(__av_value, base=0)
    return _auxiliary_vector
```

Catching with `try` is a bad idea (IMHO) because we want to know if this function raises an Exception, because it really shouldn't unless there's a change in the output from GDB.